app-admin/locksmith/locksmith-9999.ebuild

```diff
@@ -23,20 +23,18 @@ LICENSE="Apache-2.0"
 SLOT="0"
 IUSE=""
 
-DEPEND=">=dev-lang/go-1.2"
-
 src_compile() {
 	# work around gentoo hardened gcc incompatibilities with cgo
 	# see https://bugs.gentoo.org/show_bug.cgi?id=493328
 	if gcc-specs-pie; then
-		GOLDFLAGS="-extldflags -fno-PIC"
+		CGO_LDFLAGS+=" -extldflags -fno-PIC"
```

Comment:
In this case I just converted what was already there.  On reading bug 493328, there is some mention of disabling PIC, but the problem seems to be PIE, and no other ebuilds disable PIC, so I think we should remove this, then if a problem is found, disable PIC in the eclass.